README.rst

```diff
@@ -13,6 +13,9 @@ itk-jupyter-widgets
     :target: https://circleci.com/gh/InsightSoftwareConsortium/itk-jupyter-widgets
     :alt: Build status
 
+.. image:: https://mybinder.org/badge.svg
+    :target: https://mybinder.org/v2/gh/4QuantOSS/itk-jupyter-widgets/master
```

Comment:
Once this gets merged, it should possibly be:

    :target: https://mybinder.org/v2/gh/InsightSoftwareConsortium/itk-jupyter-widgets/master